disperser/apiserver/rate_config.go

```diff
@@ -10,12 +13,18 @@ const (
 	RegisteredQuorumFlagName        = "auth.registered-quorum"
 	TotalUnauthThroughputFlagName   = "auth.total-unauth-throughput"
 	PerUserUnauthThroughputFlagName = "auth.per-user-unauth-throughput"
+	TotalUnauthBlobRateFlagName     = "auth.total-unauth-blob-rate"
+	PerUserUnauthBlobRateFlagName   = "auth.per-user-unauth-blob-rate"
 	ClientIPHeaderFlagName          = "auth.client-ip-header"
+
+	blobRateMultiplier = 1e6
```

Comment:
Why do we need this?

Reply:
It's a trick for storing a fractional rate as an integer, since currently the ratelimiter accepts an integer rate type. This is probably something to clean up in the future.

Reply:
Can add a todo if you'd like.

Reply:
Add a comment will be helpful to understand, since this is taking "blobs/sec" from flags, and then storing "micro blobs/sec" in the store